lib/grape-swagger.rb

```diff
@@ -231,7 +231,8 @@ def strip_heredoc(string)
             end
 
             def parse_base_path(base_path, request)
-              (base_path.is_a?(Proc) ? base_path.call(request) : base_path) || request.base_path
+              url_scheme = request.env['rack.url_scheme']
+              (base_path.is_a?(Proc) ? base_path.call(request) : base_path) || "#{url_scheme}://#{request.env['HTTP_HOST']}#{url_scheme == 'https' ? ":80" : ''}"
```

Comment:
Line is too long. [161/80]<br>Prefer single-quoted strings inside interpolations.